src/Clock.tsx

```diff
@@ -0,0 +1,53 @@
+// import { type } from 'os';
+import React from 'react';
+
+import './App.scss';
+
+type Props = {
+  clockId: number,
+};
+
+type State = {
+  clockTime: string,
+};
+
+export class Clock extends React.Component<Props, State> {
+  state: State = {
+    clockTime: new Date().toLocaleTimeString(),
+  };
+
+  timerId: NodeJS.Timer = setInterval(() => {}, 0);
```

Comment:
```suggestion
  timerId: NodeJS.Timer | null = null;
```

Reply:
need update all logical conect of 'time'

Reply:
it's ok, you can update 💪